pyQuARC/code/custom_validator.py

```diff
@@ -102,3 +90,64 @@ def bounding_coordinate_logic_check(coordinates_dictionary):
         )
 
         return result
+
+    @staticmethod
+    def presence_check(*field_values):
```

Comment:
I think we should add some docstrings to these functions.

Reply:
added comments